notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<p>There might be an issue with calling <code>urllib.request</code> this way, but we'll double check this when we integrate your <code>requirements.txt</code> to the current <code>environment.yml</code> in <code>spacetelescope/notebooks</code> repository.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>